snippets/sass.snippets

```diff
@@ -34,3 +32,987 @@ snippet each
 snippet while
 	@while ${1:$i} ${2:>} ${3:0}
 		${0}
+snippet .
+	${1}
+		${0}
+snippet !
+	 !important
+snippet bdi:m+
+	-moz-border-image: url('${1}') ${2:0} ${3:0} ${4:0} ${5:0} ${6:stretch} ${0:stretch}
```

Comment:
Good point :) in fact they are completely useless as a handful of other ones. Just to keep this PR clear on what it does I decided to keep snippets consistent across CSS and Sass and update both of them at once in the scope of another PR. I plan to review each and every snippet: remove obsolete ones, add missing. But that should be applied to both CSS and Sass snippets together.